optimize/util/optimize-commons/src/main/java/io/camunda/optimize/service/util/configuration/security/ResponseHeadersConfiguration.java

```diff
@@ -123,12 +128,14 @@ public Map<String, String> getHeadersWithValues() {
     final Map<String, String> headers = new HashMap<>();
 
     final List<String> strictTransportSecurityHeaderValues = new ArrayList<>();
-    if (getHttpStrictTransportSecurityMaxAge() > 0) {
+    if (getHttpStrictTransportSecurityMaxAge() != null) {
```

Comment:
Checking for `> 0` and `!= null` are two different checks

The null safe check here would be something like `getHttpStrictTransportSecurityMaxAge() != null && getHttpStrictTransportSecurityMaxAge() > 0`

Reply:
But it's possible to have that header =0

Reply:
do we have to exclude 0?

Reply:
I don't think so. That was just a mistake of mine, instead of checking != null, I checked > 0

Reply:
Then this change LGTM then, I was concerned about unintended functionality change